src/components/pages/About.tsx

```diff
@@ -1,178 +1,539 @@
-                <h2 className="team-name-text">
-                    {Translate('ScientificAdvisorsAndCollaborators')}
-                </h2>
-                <div className="bio-grid-container">
-                    {renderBioBlock('Tim Evans', [Translate('Biographies', ['Tim', 'PartOne']), Translate('Biographies', ['Tim', 'PartTwo'])])}
-                    {renderBioBlock('Tyler Williamson', [Translate('Biographies', ['Tyler', 'PartOne']), Translate('Biographies', ['Tyler', 'PartTwo'])])}
-                    {renderBioBlock('Matthew Cheng', [Translate('Biographies', ['Matthew', 'PartOne']), Translate('Biographies', ['Matthew', 'PartTwo'])])}
-                    {renderBioBlock('David Naylor', [Translate('Biographies', ['DavidN', 'PartOne']), Translate('Biographies', ['DavidN', 'PartTwo'])])}
-                    {renderBioBlock('David Buckeridge', [Translate('Biographies', ['DavidB', 'PartOne']), Translate('Biographies', ['DavidB', 'PartTwo'])])}
-                    {renderBioBlock('Bruce Mazer', [Translate('Biographies', ['BruceM', 'PartOne']), Translate('Biographies', ['BruceM', 'PartTwo'])])}
-                    {renderBioBlock('Jesse Papenburg', [Translate('Biographies', ['Jesse', 'PartOne']), Translate('Biographies', ['Jesse', 'PartTwo'])])}
-                    {renderBioBlock('Jonathan Chevrier', [Translate('Biographies', ['Jonathan', 'PartOne']), Translate('Biographies', ['Jonathan', 'PartTwo'])])}
-                    {renderBioBlock('Catherine Hankins', [Translate('Biographies', ['CatherineH', 'PartOne']), Translate('Biographies', ['CatherineH', 'PartTwo'])])}
-                    {renderBioBlock('Erin O\'Connor', [Translate('Biographies', ['ErinO', 'PartOne']), Translate('Biographies', ['ErinO', 'PartTwo'])])}
-                    {renderBioBlock('Catherine Eastwood', [Translate('Biographies', ['CatherineE', 'PartOne']), Translate('Biographies', ['CatherineE', 'PartTwo'])])}
-                </div>
+      <div>
+        <b>{name}</b>
+        <br />
+        {description.map((line) => {
+          return (
+            <div key={line}>
+              {line}
+              <br />
             </div>
+          );
+        })}
+        {linkedIn ? (
+          <a
+            href={linkedIn}
+            onClick={() => selectItem("LinkedIn", name)}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <FontAwesomeIcon icon={faLinkedin} className="mr-2 linked-in" />
+          </a>
+        ) : null}
+        {twitter ? (
+          <a
+            href={twitter}
+            onClick={() => selectItem("Twitter", name)}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <FontAwesomeIcon icon={faTwitter} className="mr-2 twitter" />
+          </a>
+        ) : null}
+        {email ? (
+          <a href={`mailto:${email}`} onClick={() => selectItem("Email", name)}>
+            <FontAwesomeIcon icon={faEnvelope} className="mr-2 email" />
```

Comment:
Changed all of these `mr-2` instead, since there was weird spacing if only the email / twitter icon was rendered. This will make sure spacing is correct regardless of number of icons.